src/main/java/org/openrewrite/java/testing/jmockit/JMockitExpectationsToMockito.java

```diff
@@ -213,5 +267,17 @@ private static String getMockitoStatementTemplate(Expression result) {
             }
             return template;
         }
+
+        private static class ArgumentMatcherTemplateInfo {
```

Comment:
Feel free to use `@Value` on this class; eventually we'll convert those to records directly.

Reply:
Sounds good! I read that we should only assume Java 17 for tests and Java 8 for recipes so I did not use Records here.